flink-contrib/flink-storm-compatibility/flink-storm-compatibility-core/src/main/java/org/apache/flink/stormcompatibility/wrappers/AbstractStormSpoutWrapper.java

```diff
@@ -97,9 +102,20 @@ public AbstractStormSpoutWrapper(final IRichSpout spout,
 	}
 
 	@Override
+	public void open(Configuration parameters) throws Exception {
+		config = new HashMap();
+
+		/* parameters is task configuration, we can get storm configuration only from job configuration */
+		Map stormConf = StormWrapperSetupHelper.getStormConfFromContext(super.getRuntimeContext());
+		if (stormConf != null) {
+			config.putAll(stormConf);
+		}
+	}
```

Comment:
I just had a closer look at the code. Because we do not use the given `Configuration`, we should move the code into `run` method and remove the overwrite of `open()`.